extensions/shared/blocks/cover/editor.scss

```diff
@@ -0,0 +1,13 @@
+.jetpack-cover-media-placeholder {
+	width: 100%;
+
+	// tweak Upgrade nudge.
```

Comment:
Are you able to include the way in which we we "tweaking"?

Reply:
We are going to update this pretty soon since we need to apply similar changes to the video block. Can we kick it for a next PR?